crates/bevy_render/src/gpu_readback.rs

```diff
@@ -349,7 +349,7 @@ fn map_buffers(mut readbacks: ResMut<GpuReadbacks>) {
 // Utils
 
 pub(crate) fn align_byte_size(value: u32) -> u32 {
```

Comment:
Can you add basic doc strings to this function? It's hard to validate that this is doing what it says it should without them :)

Reply:
Done. I've also tried to divine what the other util methods do.